packages/compilers/package.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
We should also setup running the tests in the debugger with .vscode configuration